release-notes/20230330-recover-broker-info.md

```diff
@@ -0,0 +1,2 @@
+<!-- markdownlint-disable MD041 -->
+A new `subctl recover-broker-info` command is added that recovers a lost `broker-info.subm` file.
```

Comment:
```suggestion
Added a `subctl recover-broker-info` command to recover a lost `broker-info.subm` file.
```

Reply:
(this is passive voice, see https://www.hamilton.edu/academics/centers/writing/seven-sins-of-writing/1)

Reply:
there is just 1 `broker-info.subm` file that is recovered. So "file". Changed the sentence to passive voice.